content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+{{ blockEnd }}
+
+{{ startTab "Javascript/Typescript"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Python"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Rust"}}
+
+**SDK: Outbound HTTP**
+
+Local: Yes
+
+Cloud: Yes
+
+Documentation: [https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests](https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests)
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http](https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http)
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local: Yes
+
+Cloud: Soon
+
+Documentation: [https://developer.fermyon.com/spin/kv-store.md](https://developer.fermyon.com/spin/kv-store.md)
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/rust-key-value](https://github.com/fermyon/spin/tree/main/examples/rust-key-value)
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local: Yes
+
+Cloud: Yes (non-hosted)
```

Comment:
I'm not sure what this is saying.  MySQL is always non-hosted, whether local or cloud; phrasing it this way suggests that local _is_ hosted.